src/main.js

```diff
@@ -237,6 +237,8 @@ let load = base.loader({
         regionResolver:   ctx.regionResolver,
         publicProxies:    ctx.cfg.app.publicArtifactBucketProxies,
         credentials:      ctx.cfg.taskcluster.credentials,
+        cloudMirrorHost:  ctx.cfg.app.cloudMirrorHost,
+        artifactRegion:   ctx.cfg.aws.region,
```

Comment:
Please label this as a hack that will be removed!

At-least I strongly suspect we should...